BugSplat.uplugin

```diff
@@ -28,11 +28,8 @@
 		}
 	],
 	"PostBuildSteps": {
-		"Win64": [
-			"call \"$(ProjectDir)\\Plugins\\BugSplat\\Source\\Scripts\\BugSplat.bat\" $(TargetPlatform)"
```

Comment:
Why did this go away? I believe we need this to upload Win64 symbols for Desktop.

Reply:
Apparently, the [`BugSplat.bat` file](https://github.com/BugSplat-Git/bugsplat-unreal/blob/main/Source/Scripts/BugSplat.bat) doesn't do anything atm, so I've removed it just to get rid of the message about misconfigured plugin during the build.

Reply:
I've just figured out that this .bat-file can be modified via the editor menu and store the actual command for sym uploading. I'll put it back!